### ISSUE TITLE
Getting "update check failed" error even though I have access to ~/.config

### ISSUE THREAD

**Issue by eyalch** [dotenvx/dotenvx#150]
I'm getting the following error message even though my user does have access to `~/.config`:

```sh
$ npx dotenvx run -- echo foo
[dotenvx@0.27.1] injecting env (2) from .env
foo
┌─────────────────────────────────────────────────────────┐
│          @dotenvx/dotenvx update check failed           │
│           Try running with sudo or get access           │
│          to the local update config store via           │
│ sudo chown -R $USER:$(id -gn $USER) /Users/eyal/.config │
└─────────────────────────────────────────────────────────┘
```

After a bit more debugging, I found that the owner of the `~/.config/configstore` directory (which `configstore` uses) is `root`. I have no idea how this happened, but the error message printed by `@dotenvx/dotenvx` isn't helpful and also slightly misleading, since I do have access to `~/.config`.

As this case is very rare, should we consider printing the error that `configstore` throws? Or even simpler, remove the error handling?

Are there any legit cases where you expect the user to _not_ own `~/.config`, in which case the original message _is_ helpful?

Here's the relevant piece of code:
https://github.com/dotenvx/dotenvx/blob/01378effdcb819c6d2d05e132a937c05aa329588/src/lib/helpers/updateNotifier.js#L49-L63

**Comment by motdotla** [comment]
thank you. i will get this fixed up today. sorry about this!

**Comment by motdotla** [comment]
this is a limitation of configstore - affects some people's machines. we're moving all settings to `conf`. then this will be patched up.

https://github.com/sindresorhus/env-paths?tab=readme-ov-file#pathsconfig

**Comment by eyalch** [comment]
I'm not sure if this applies to my case. As I mentioned, my user _does_ have access to `~/.config` and always has access. The access issue was with `~/.config/configstore`.

**Comment by eyalch** [comment]
My point is that the error message printed by dotenvx incorrectly assumes that my user has no access to `~/.config`, which isn't the case.

**Comment by motdotla** [comment]
effort happening here: https://github.com/dotenvx/dotenvx/pull/151

**Comment by motdotla** [comment]
@eyalch possibly, nonetheless, i want to simplify to a single config store - `conf`, and put all settings in there. this will give the dev one single place to go to change all these things (for example to turn updates off). i want them to have ease of control as easy as editing a .env file. meta.

**Comment by motdotla** [comment]
didn't quite finish today but will wrap up tomorrow. almost there.

**Comment by motdotla** [comment]
we just released `v0.27.2` which should fix this problem. give it a try.
